tools/audio/Makefile

```diff
@@ -1,14 +1,44 @@
+PROGRAMS := sbc
+
+ifeq ($(shell which xml2-config),)
+  $(error xml2-config not found. Did you install libxml2-dev?)
+endif
+
+CLANG_FORMAT := clang-format-14
+FORMAT_ARGS := -i -style=file
+
+CC := gcc
+CFLAGS := -Wall -Wextra -pedantic
+OPTFLAGS := -Og -g3
```

Comment:
```suggestion
OPTFLAGS := -O2 -g3
```

Reply:
I plan to boost optimization later once all the tools are present